Kernel/GlobalProcessExposed.cpp

```diff
@@ -100,6 +100,34 @@ class ProcFSARP final : public ProcFSGlobalInformation {
     }
 };
 
+class ProcFSRoute final : public ProcFSGlobalInformation {
+public:
+    static NonnullRefPtr<ProcFSRoute> must_create();
+
+private:
+    ProcFSRoute();
+    virtual ErrorOr<void> try_generate(KBufferBuilder& builder) override
+    {
+        auto array = TRY(JsonArraySerializer<>::try_create(builder));
+        TRY(routing_table().with([&](auto const& table) -> ErrorOr<void> {
+            for (auto& it : table) {
+                auto obj = TRY(array.add_object());
+                auto destination = it.destination.to_string().release_value_but_fixme_should_propagate_errors();
```

Comment:
any reason we can't TRY() these string copies? seems to me we're in a context return ErrorOr<void> so it should work?

Reply:
Hmm. I suppose there's no reason we can't if we'd rather `TRY()` than `VERIFY()` them and drop the `release_value_but_fixme_should_propagate_errors`. 

I was just following the lead from the other existing proc entries, so maybe if we want to make that change it merits a separate PR for all the proc entries together?

Reply:
Cheers!